addon/validations/construction-project.js

```diff
@@ -43,101 +42,7 @@ export default {
     validateNumber({ integer: true }),
   ],
   typeOfConstruction: [validatePresence(true)],
-  projectAnnouncementDate: [
-    validatePresence(true),
-    validateDate({
-      onOrAfter: new Date("2000-01-01"),
-      message: "{description} must be on or after {onOrAfter}",
-    }),
-    validateDate({
-      onOrBefore: new Date(),
-      message: "{description} must be on or before {onOrBefore}",
-    }),
-  ],
-  buildingPermitIssueDate: [
-    validateDate({
-      onOrAfter: new Date("2000-01-01"),
-      message: "{description} must be on or after {onOrAfter}",
-      allowBlank: true,
-    }),
-    validateDate({
-      onOrBefore: new Date(),
-      message: "{description} must be on or before {onOrBefore}",
-      allowBlank: true,
-    }),
-  ],
-  withdrawalDate: [
-    validateDate({
-      onOrAfter: new Date("2000-01-01"),
-      message: "{description} must be on or after {onOrAfter}",
-      allowBlank: true,
-    }),
-    validateDate({
-      onOrBefore: new Date(),
-      message: "{description} must be on or before {onOrBefore}",
-      allowBlank: true,
-    }),
-  ],
-  projectStartDate: [
-    validateDate({
-      onOrAfter: new Date("2000-01-01"),
-      message: "{description} must be on or after {onOrAfter}",
-      allowBlank: true,
-    }),
-    validateDate({
-      onOrBefore: new Date(),
-      message: "{description} must be on or before {onOrBefore}",
-      allowBlank: true,
-    }),
-  ],
-  constructionAuthorisationDeniedDate: [
-    validateDate({
-      onOrAfter: new Date("2000-01-01"),
-      message: "{description} must be on or after {onOrAfter}",
-      allowBlank: true,
-    }),
-    validateDate({
-      onOrBefore: new Date(),
-      message: "{description} must be on or before {onOrBefore}",
-      allowBlank: true,
-    }),
-  ],
-  cancellationDate: [
-    validateDate({
-      onOrAfter: new Date("2000-01-01"),
-      message: "{description} must be on or after {onOrAfter}",
-      allowBlank: true,
-    }),
-    validateDate({
-      onOrBefore: new Date(),
-      message: "{description} must be on or before {onOrBefore}",
-      allowBlank: true,
-    }),
-  ],
-  projectCompletionDate: [
-    validateDate({
-      onOrAfter: new Date("2000-01-01"),
-      message: "{description} must be on or after {onOrAfter}",
-      allowBlank: true,
-    }),
-    validateDate({
-      onOrBefore: new Date(),
-      message: "{description} must be on or before {onOrBefore}",
-      allowBlank: true,
-    }),
-  ],
-  projectSuspensionDate: [
-    validateDate({
-      onOrAfter: new Date("2000-01-01"),
-      message: "{description} must be on or after {onOrAfter}",
-      allowBlank: true,
-    }),
-    validateDate({
-      onOrBefore: new Date(),
-      message: "{description} must be on or before {onOrBefore}",
-      allowBlank: true,
-    }),
-  ],
+  projectAnnouncementDate: [validatePresence(true)],
```

Comment:
Did you check that all removed validations are either covered by the quality rules or are no longer needed?

Reply:
Noooo :point_right: :point_left: 
Will do that.

Reply:
Tested, results in open issues on the project

Reply:
Nice!